descwl_coadd/coadd.py

```diff
@@ -1247,8 +708,9 @@ def _get_jac(self, *, cenx, ceny):
             Center for the output ngmix jacobian (not place of evaluation)
         """
 
-        coadd_cen = self.coadd_wcs.getPixelOrigin()
-        dm_jac = self.coadd_wcs.linearizePixelToSky(coadd_cen, geom.arcseconds)
+        coadd_wcs = self.coadd_exp.getWcs()
+        coadd_cen = coadd_wcs.getPixelOrigin()
+        dm_jac = coadd_wcs.linearizePixelToSky(coadd_cen, geom.arcseconds)
```

Comment:
If I understand it correctly, the coadd is a small subset of a larger WCS. This routine appears to return the jacobian at the center of the larger WCS, but we want it at the center of the smaller coadd within the larger system.